en/docs/learn/tenant-wise-email-sender-configuration.md

```diff
@@ -99,31 +101,33 @@ We are configuring email-sender configurations by adding the following configura
     
     Following is a sample configuration for `EmailPublisher.xml` file. 
     
-    ```xml
-    <?xml version="1.0" encoding="UTF-8"?>
-    <eventPublisher name="EmailPublisher" statistics="disable"
-      trace="disable" xmlns="http://wso2.org/carbon/eventpublisher">
-      <from streamName="id_gov_notify_stream" version="1.0.0"/>
-      <mapping customMapping="enable" type="text">
-        <inline>{{body}}{{footer}}</inline>
-      </mapping>
-      <to eventAdapterType="email">
-        <property name="email.address">{{send-to}}</property>
-        <property name="email.type">{{content-type}}</property>
-        <property name="email.subject">{{subject}}</property>
-        <property name="mail.smtp.password">xxxxx</property>
-        <property name="mail.smtp.from">resourcesiam@gmail.com</property>
-        <property name="mail.smtp.user">resourcesiam</property>
-      </to>
-    </eventPublisher>
-    ```
+    ??? info "Sample Email Publisher"
+        ```
+        <?xml version="1.0" encoding="UTF-8"?>
+        <eventPublisher name="EmailPublisher" statistics="disable"
+          trace="disable" xmlns="http://wso2.org/carbon/eventpublisher">
+          <from streamName="id_gov_notify_stream" version="1.0.0"/>
+          <mapping customMapping="enable" type="text">
+            <inline>{{body}}{{footer}}</inline>
+          </mapping>
+          <to eventAdapterType="email">
+            <property name="email.address">{{send-to}}</property>
+            <property name="email.type">{{content-type}}</property>
+            <property name="email.subject">{{subject}}</property>
+            <property name="mail.smtp.password">xxxxx</property>
+            <property name="mail.smtp.from">resourcesiam@gmail.com</property>
+            <property name="mail.smtp.user">resourcesiam</property>
+          </to>
+        </eventPublisher>
+        ``` 
+        
     !!! note
         You do not need to configure all the configurable parameters. If you havent configured in the `EmailPublisher.xml`,
         Configurations in the `output-event-adapters.xml` will be used for unconfigured parameters. 
     
 5. Configure tenant loading and unloading for your tenant. [Configuring the Tenant Loading Policy](../../administer/configuring-the-tenant-loading-policy) 
 
-6. These configurations will apply to the tenant loading process.
+6. These configurations will apply on the tenant loading process.
```

Comment:
"...will apply to the tenant loading process" OR "These configurations will be applicable during the tenant loading process. "

This sentence does not need to be a step since its not an action that the user needs to do

Reply:
fixed